wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -915,6 +916,11 @@ class KotlinGenerator private constructor(
             if (constant.documentation().isNotBlank()) {
               addKdoc("%L\n", constant.documentation())
             }
+            if ("true" == constant.options().get(ENUM_DEPRECATED)) {
```

Comment:
nit: I don't like Yoda statements
why not `if (constant.options().get(ENUM_DEPRECATED) == "true")`